modules/jupyternotebook/parametrize.nf

```diff
@@ -0,0 +1 @@
+../rmarkdown/parametrize.nf
```

Comment:
TODO: replace with the actual file... just keeping the symlink for development such that the files don't get out-of-sync.

Reply:
I know it's already consistent in what's already imported in the modules, but shouldn't it be `paramet**e**rize.nf` (or even better `parameterise.bf` 💂 ;) )